common/scala/src/main/resources/application.conf

```diff
@@ -114,6 +114,14 @@ whisk {
         actions-ddoc = "whisks.v2.1.0"
         activations-ddoc = "whisks.v2.1.0"
         activations-filter-ddoc = "whisks-filters.v2.1.0"
+
+        # Size limit for inlined attachments. Attachments having size less than this would
+        # be inlined with there content encoded in attachmentName
+        max-inline-size = 16 k
```

Comment:
can this be set per deployment?

Reply:
It should be possible to override the default via env variable `CONFIG_whisk_db_maxInlineSize`. Is that what you meant?

Reply:
Yes - great.